src/Funds/FundDetails/FundCurrentBudget/FundCurrentBudgetContainer.js

```diff
@@ -1,55 +1,28 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import ReactRouterPropTypes from 'react-router-prop-types';
 import { FormattedMessage } from 'react-intl';
 
 import { stripesConnect } from '@folio/stripes/core';
-import {
-  Accordion,
-  Button,
-  Icon,
-} from '@folio/stripes/components';
-import {
-  useShowToast,
-} from '@folio/stripes-acq-components';
+import { Button } from '@folio/stripes/components';
 
-import { budgetsResource } from '../../../common/resources';
 import FundBudgets from '../FundBudgets';
+import {
+  BUDGET_ROUTE,
+  BUDGET_VIEW_ROUTE,
+} from '../../../common/const';
 import { BUDGET_STATUSES } from '../../../components/Budget/constants';
 import { SECTIONS_FUND } from '../../constants';
 
 const FundCurrentBudgetContainer = ({
+  budget,
   currency,
-  currentFY,
-  fundId,
   history,
-  mutator,
   openNewBudgetModal,
 }) => {
-  const [isLoading, setIsLoading] = useState(false);
-  const [budget, setBudget] = useState([]);
-  const showToast = useShowToast();
-
-  useEffect(() => {
```

Comment:
Could you please describe why any changes are required to FundCurrentBudgetContainer.js  ? From my point of view button usually lives somewhere in fundetails (actions)

Reply:
The idea is not to fetch the current budget twice.